demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -287,6 +296,7 @@ export function ProductsGrid() {
                 components={{
                     Toolbar: ProductsGridToolbar,
                 }}
+                columnVisibilityModel={{ manufacturer: false }}
```

Comment:
can't the default visibility also specified using visible: false on the column config?

Reply:
@jamesricky is this something we should support?

Reply:
Why would you hide a column completely? 🤔 
Should this really only be visible, once the user explicitly shows this column? 
I feel like most users won't actually realize they can show this data, when they can't actually see the column so I'm not sure this will be a very common use case. 

But if it is we can of course add the ability to set this to `false`.

Reply:
> Why would you hide a column completely?

- if you need only the filter
- if you need only to excel export the column
- if only some users need the column

...we definitely need a way to set visible to false